content/themis/languages/python/installation.md

```diff
@@ -75,13 +75,151 @@ Once PyThemis is installed, you can [try out code examples](../examples/).
 If the stable package version does not suit your needs,
 you can manually build and install the latest version of Themis from source code.
 
+ 1. [Build and install Themis Core library](/themis/installation/installation-from-sources/)
+    into your system.
+
+ 2. Install Python dependencies
+
+    Depending on how you want to install PyThemis, there are different requirements.
+
+    To create a `.whl` package for installation inside virtualenv, you'll need `setuptools` and `wheel`
+
+    * Install globally
+
+      For Debian, Ubuntu:
+
+      ```bash
+      sudo apt install python3-setuptools python3-wheel
+      ```
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      sudo yum install python3-setuptools python3-wheel
+      ```
+-->
+
+    * Or install into virtualenv
```

Comment:
You can use markdown viewer/editor in the pycharm or idea, or you can open .md file in the github to look how it will be rendered. We need valid markdown due to after that it will be rendered with hugo server that has own markdown parser. 
![image](https://github.com/cossacklabs/product-docs/assets/6237700/bbf48d73-6078-44f5-8f15-7ed67227ee49)

Reply:
It's GitHub that renders like this. When running Hugo locally, it renders properly in web browser